src/components/Settings/GroupLink.js

```diff
@@ -0,0 +1,77 @@
+import React, { useRef, useState } from "react";
+import Button from "@material-ui/core/Button";
+import Dialog from "@material-ui/core/Dialog";
+import DialogActions from "@material-ui/core/DialogActions";
+import DialogContent from "@material-ui/core/DialogContent";
+import DialogContentText from "@material-ui/core/DialogContentText";
+import DialogTitle from "@material-ui/core/DialogTitle";
+import { makeStyles, Typography } from "@material-ui/core";
+import Snackbar from "../Snackbar";
+
+const useStyles = makeStyles((theme) => ({
+  buttons: {
+    display: "flex",
+    justifyContent: "space-between",
+    padding: 25,
+    paddingTop: 20,
+  },
+  button: {
+    paddingLeft: 0,
+    paddingRight: 0,
+  },
+  groupID: {
+    fontWeight: 500,
```

Comment:
```suggestion
    fontWeight: theme.typography.fontWeightMedium,
```
Kleine aber feine Änderung, die spätere Änderungen vereinfachen würde :)